test/test_common/utility.h

```diff
@@ -75,6 +75,14 @@ namespace Envoy {
     EXPECT_DEATH(statement, message);                                                              \
   } while (false)
 
+#define VERIFY_ASSERTION(statement)                                                                \
+  {                                                                                                \
```

Comment:
do { ... } while(false) as in the other macros. (This will force VERIFY_ASSERTION() to be followed by a semicolon in all cases, which I think we want.)